models/accessom3.py

```diff
@@ -0,0 +1,90 @@
+"""Specific Access-OM3 Model setup and post-processing"""
+
+from collections import defaultdict
+import re
+from pathlib import Path
+from payu.models.cesm_cmeps import Runconfig
```

Comment:
Sorry, it only just occurred to me that importing payu might run into import errors in `access-om2-configs` QA tests CI https://github.com/ACCESS-NRI/access-om2-configs/blob/a57e7fd10c351edef7427ddf42c71ce53c24df27/.github/workflows/pr-1-ci.yml#L114-L143, as `payu` is not included `requirements.txt`. It would work ok on NCI for the repro tests, as they use payu conda environment in vk83. @aidanheerdegen we talked about earlier today about using separate python virtual environments on NCI to be able to install different versions of `model-config-tests` for potentially concurrent repro tests. Should a version of payu be included in that so the environment is separate from the payu conda environments and payu can be included in requirements.txt?

Reply:
Ah I (mis)read [this](https://github.com/ACCESS-NRI/model-config-tests/blob/b086103bb4a2ff87f7d6bb82cd767ab173b1e089/models/accessom2.py#L3-L4) to mean that Payu would always be available.

Reply:
Yeah sorry about that, I've included payu as a dependency so it'll now be available